tests/integ/test_context.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
+#
+
+from snowflake.snowpark.context import get_active_session
+
+
+def test_get_active_session(session):
```

Comment:
I considered adding some negative tests for no active sessions and for multiple sessions but they are already present in `tests/integ/test_session.py`.